data/helpers.d/utils

```diff
@@ -112,7 +112,7 @@ ynh_backup_before_upgrade () {
 #
 # Details:
 # This helper downloads sources from SOURCE_URL if there is no local source
-# archive in /opt/yunohost-apps-src/APP_ID/SOURCE_FILENAME
+# archive in /var/cache/yunohost/ynh_setup_source/APP_ID/SOURCE_FILENAME
```

Comment:
/opt/yunohost-apps-src/ isn't a cache for package.
It's a way to allow users to install a package without an internet connection.

So that you can't just replace it.

Reply:
Yep, this feature may (might?) be used by some project in Africa/Asia with low/no connectivity.

Reply:
Someone with low/no connectivity will probably want to go into /var/cache/apt/archives/ for debian packages. Many of our packages won't work only without debian packages:https://github.com/search?q=org%3AYunoHost-Apps+ynh_install_app_dependencies&type=Code

I'm unsure what's the issue here, isn't this PR an improvement in this low/no internet scenario?

Do we need to add a readme file telling where to find the new folder?
Btw, this folder is not written in our documentation: https://github.com/YunoHost/doc/search?q=yunohost-apps-src&type=Issues

Reply:
I agree with the idea to change the location. I think, the cache should be manage correctly if the partition behind /var/cache is quite full